packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -416,6 +416,7 @@ const QueryEditorHeaderContent = observer(
           </div>
         ) : (
           <div
+            onDoubleClick={updateQuery}
```

Comment:
nice! But in terms of UX, we need visual hints that this is possible, using tooltip or some CSS border, or altering the `cursor: pointer`